cmd/root.go

```diff
@@ -51,6 +54,11 @@ func initConfig() {
 		viper.SetConfigName(".trento")
 	}
 
+	if logLevel == "" {
```

Comment:
As commented before, this looks strange as we should use the default value and not this comparison